app/javascript/admin/controllers/config_controller.js

```diff
@@ -16,6 +16,8 @@ const emailAuthModalBody = `
   <p>If you disable Email address as a registration option, people cannot create an account with their email address.</p>
   <p>However, people who have already created an account using their email address can continue to login.</p>`;
 
+// Note: In an effort to move away from Stimulus and create consistency across the codebase
```

Comment:
<3 

Also, totally tangential suggestion. It's customary to capitalize NOTE, TODO, FIXME, etc. because many/most coding editors will specifically highlight those and/or have extensions for treating them like todo lists, etc.